scripts/backup

```diff
@@ -77,11 +70,7 @@ ynh_backup "/etc/cron.d/$app"
 # BACKUP THE DATA DIRECTORY
 #=================================================
 
-backup_core_only=$(ynh_app_setting_get "$app" backup_core_only)
-# If backup_core_only have any value in the settings.yml file, do not backup the data directory
-if [ -z $backup_core_only ]
-then
-	ynh_backup "/home/yunohost.app/${app}/data"
-else
-	echo "Data dir will not be saved, because backup_core_only is set." >&2
-fi
+# The 1 parameter indicates the directory is "big",
+# so that it won't be backed up before upgrade
+# This argument has to be the third one.
+ynh_backup "/home/yunohost.app/${app}/data" "/home/yunohost.app/${app}/data" 1
```

Comment:
This part should be really tested, as well for Jirafeau.
I wonder what will going on with a restoration for which a directory wasn't backup.
If the directory is simply missing, it could be a real problem.

Reply:
Hm, how is it different from the situation without this helper?
We only omit backing up this directory when upgrading, and the restore script is adapted to that. How can that fail...?

Reply:
That's indeed in that restore script.
Maybe I was thinking especially about Jirafeau, don't remember...

Reply:
Confirm that it doesn't work for Jirafeau, and not_mandatory isn't yet available for restore helper